app/connectors/PensionsConnector.scala

```diff
@@ -77,6 +78,27 @@ class PensionsConnector @Inject() (val http: HttpClient, val appConfig: AppConfi
     EitherT(res)
   }
 
+  def getUkPensionIncome(nino: Nino, taxYear: TaxYear)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext): ApiResultT[Option[IncomeFromPensionsViewModel]] = {
+    val url = appConfig.upPensionIncomeAnswersUrl(taxYear, nino)
```

Comment:
should be 'ukPensionIncomeUrl' not 'upPension...'

Reply:
good idea, will leave it as it is for now